compiler-rt/lib/mpk_untrusted/tests/util/mpk_untrusted_test_config.h

```diff
@@ -1,4 +1,4 @@
-//===-- mpk_untrusted_test_config.h ------------------------------*- C++ -*-===//
+//===-- mpk_untrusted_test_config.h -------------------------------- C++ --===//
```

Comment:
nit: those '*' characters are used for editor config. Editors like vim and emacs know how to process those directives for syntax highlighting purposes.